internal/restinterface/internalhandler/internalhandler.go

```diff
@@ -134,7 +134,9 @@ func (h *Handler) SetCertificateFilePath(path string) {
 
 // APIV1Ping handles ping request from remote orchestration
 func (h *Handler) APIV1Ping(w http.ResponseWriter, r *http.Request) {
-	h.helper.Response(w, nil, http.StatusOK)
+	var responseBytes []byte
+	responseBytes = []byte("Pong")
```

Comment:
```suggestion
	responseBytes = []byte("Pong\n")
```
How about adding `\n`?

Reply:
done